src/prefabs/structures/Box/index.ts

```diff
@@ -0,0 +1,16 @@
+import { component, PrefabComponentOption } from '@betty-blocks/component-sdk';
+import { PrefabReference } from '@betty-blocks/component-sdk/build/prefabs/types/component';
```

Comment:
oh nooos, this type should be exported/imported at the top level.

Reply:
you know this happens at other places too already xD